chainer_chemistry/datasets/numpy_tuple_dataset.py

```diff
@@ -48,6 +49,10 @@ def __len__(self):
     def get_datasets(self):
         return self._datasets
 
+    @property
+    def converter(self):
+        return concat_mols
```

Comment:
TODO later: consider design. NumpyTupleDataset may be used with other converter.